starlite/testing/test_client/client.py

```diff
@@ -124,11 +145,13 @@ def session(self) -> "CookieBackend":
             "To access the session backend directly, use the session_backend attribute",
             PendingDeprecationWarning,
```

Comment:
I know this is before you, hut why are we deprecating this?

Reply:
Hm, idk. You suggested changing the name from `session` to `session_backend` at some point, so I made `session` only return a `CookieBackend` and deprecate the method. We could remove the deprecation as well, but I think in the long run, having only the `session_backend` does make more sense, since it's "correcter". It *does* return a session backend, and not a session.

Reply:
Sure, I didn't remember. Let's set a version for dropping this? Say 1.40?

Reply:
Shouldn't this only happen in a major version since we're following semver?

Reply:
we can do this - but we are not going for a major version until pydantic 2.0 is out i think.

Reply:
Well I don't mind not following semver. It's not the end-all be-all.